packages/cli/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "bin": {
+    "sequelize": "./bin/run.js"
+  },
+  "bugs": "https://github.com/sequelize/sequelize/issues",
+  "dependencies": {
+    "@oclif/core": "^3.26.0",
+    "@oclif/plugin-help": "^6.0.20",
+    "@sequelize/utils": "workspace:*",
+    "@types/inquirer": "^9.0.7",
+    "chalk": "^5.3.0",
+    "cosmiconfig": "^9.0.0",
+    "inquirer": "^9.2.14",
+    "zod": "^3.22.4"
+  },
+  "description": "The Sequelize CLI\nDocumentation: https://sequelize.org/docs/v7/cli/",
+  "devDependencies": {
+    "@oclif/test": "3.2.8",
+    "@types/chai": "4.3.14",
+    "@types/mocha": "10.0.6",
+    "chai": "4.4.1",
+    "concurrently": "8.2.2",
+    "mocha": "10.3.0",
+    "oclif": "4.6.3",
+    "typescript": "5.4.3"
+  },
+  "engines": {
+    "node": ">=18.0.0"
+  },
+  "exports": "./lib/index.js",
+  "files": [
+    "/bin",
+    "/lib",
+    "/static",
+    "/oclif.manifest.json"
+  ],
+  "homepage": "https://sequelize.org/docs/v7/cli/",
+  "keywords": [],
+  "license": "MIT",
+  "main": "lib/index.js",
+  "name": "@sequelize/cli",
+  "oclif": {
+    "bin": "sequelize",
+    "dirname": "sequelize",
+    "commands": "./lib/commands",
+    "plugins": [
+      "@oclif/plugin-help"
+    ],
+    "topicSeparator": " ",
+    "topics": {
+      "migration": {
+        "description": "Commands for managing database migrations"
+      }
+    }
+  },
+  "repository": "https://github.com/sequelize/sequelize",
+  "scripts": {
+    "build": "rm -rf lib && tsc --project tsconfig.build.json",
+    "postpack": "rm -f oclif.manifest.json",
```

Comment:
I was testing this on my Windows machine as well, `rm -rf` does not work there. Not sure why this is not something picked up by the CI. Maybe there's a difference between Windows 11 and Windows Server?

Reply:
It's not picked up by the CI because we don't release from windows and we only pack as part of the release process